common/components/utils/guard.js

```diff
@@ -3,7 +3,7 @@
 import _ from 'lodash';
 
 class Guard {
-  static click(func: string, duration: number, options: object): string {
+  static click(func: string, duration = 1000: number, options = {'leading': true}: object): string {
```

Comment:
I added these if we want to just write `Guard.click(function)` in a given component rather than specifying out duration and options each time. I think these are reasonable enough default values for our purposes.

Reply:
Sounds reasonable.  My only comment is that this function as laid out could be used for things besides clicking, so we might want to rename it to something more generic.

Reply:
Renamed it to Guard.duplicateInput - so it reads as "guard (against) duplicate inputs," which is the most generic name for it I can think of. If something better comes to mind it's an easy fix right now.